index.js

```diff
@@ -269,6 +270,9 @@ async function run() {
     const forceNewDeployInput = core.getInput('force-new-deployment', { required: false }) || 'false';
     const forceNewDeployment = forceNewDeployInput.toLowerCase() === 'true';
 
+    let desiredCount = parseInt((core.getInput('desired-count', {required: false}))) || 0;
```

Comment:
Can we remove "|| 0"? I'd rather to keep the desire count to be the current task num instead of setting it to 0, which can cause a problem when the service was scaled up.